doc/dev_guide/internals/box_protocol.rst

```diff
@@ -1273,16 +1274,36 @@ The fields within IPROTO_BALLOT are map items:
 
 ..  code-block:: none
 
-    IPROTO_BALLOT_IS_RO (0x01) + MP_BOOL
+    IPROTO_BALLOT_IS_RO_CFG (0x01) + MP_BOOL
     IPROTO_BALLOT_VCLOCK (0x02) + vclock
     IPROTO_BALLOT_GC_VCLOCK (0x03) + vclock
-    IPROTO_BALLOT_IS_LOADING (0x04) + MP_BOOL
+    IPROTO_BALLOT_IS_RO (0x04) + MP_BOOL
     IPROTO_BALLOT_IS_ANON = 0x05 + MP_BOOL
+    IPROTO_BALLOT_IS_BOOTED = 0x06 + MP_BOOL
 
-IPROTO_BALLOt-IS_ANON corresponds to :ref:`box.cfg.replication_anon <cfg_replication-replication_anon>`.
+IPROTO_BALLOT_IS_RO_CFG and IPRO_BALLOT_VCLOCK and IPROTO_BALLOT_GC_VCLOCK and IPROTO_BALLOT_IS_RO
+were added in version :doc:`2.6.1 </release/2.6.1>`.
+IPROTO_BALLOT_IS_ANON was added in version :doc:`2.7.1 </release/2.7.1>`.
+IPROTO_BALLOT_IS_BOOTED was added in version 2.7.3 and 2.8.2 and 2.9.1.
```

Comment:
These versions are not released yet.

Reply:
It's an improvement to change "IPROTO_BALLOT_GC_VCLOCK may have the value" to "IPROTO_BALLOT_GC_VCLOCK can be the clock value", yes.

It's true that "These versions are not released yet." In fact the changes have been done already. However, in https://github.com/tarantool/doc/issues/2185, in response to the question "@Gerold103 Since which Tarantool version we have these changes?", the answer was "2.9.1, 2.8.2, 2.7.3". I understood this to mean that these are the versions that we should mention.